array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,12 @@
+/* exported compact */
+function compact(array) {
+  var arr = array;
+  var newArray = [];
+  for (var i = 0; i < arr.length; i++) {
+    if (!arr[i]) {
+      continue;
+    }
+    newArray.push(arr[i]);
```

Comment:
Usually when the keyword `continue` comes up in code, it's an indicator that your logic is backwards. In this case, the real work is to add the value to the array, so a truthy value in `arr[i]` should be what your condition is looking for.